blog/admin.py

```diff
@@ -0,0 +1,9 @@
+from django.contrib import admin
+
+from blog.models import Post
+
+
+@admin.register(Post)
+class PostAdmin(admin.ModelAdmin):
+    list_display = ["title", "published_date"]
+    raw_id_fields = ["author"]
```

Comment:
I had no idea about `raw_id_fields` and now there is a project I need to go revisit so I can implement this!